src/test/java/utilities/Util.java

```diff
@@ -13,4 +21,20 @@ public static void wait(int seconds) {
 		}
 	}
 
-}
+
+	/*
+	 * moveToElement
+	 * public static void  moveToElement() throws InterruptedException {
+		System.setProperty("webdriver.chrome,driver", "src/test/resources/chromedriver");
+		WebDriver driver = new ChromeDriver();
+		driver.get("https://demoqa.com/droppable");
+		driver.manage().window().maximize();
+		
+		Actions actions = new Actions(driver);
+		
+		WebElement targetLocation = driver.findElement(By.id("toolTipButton"));
+		Thread.sleep(3000);
+		actions.moveToElement(targetLocation).build().perform();
+	 * @Author: Mustafa W
+	 */
```

Comment:
1, Is the function done or no? It's commented
2, To keep the function reusable, it should take a parameter
3, Not sure why the files under `target/` folder is checked in, those should be ignored
4, Fix formatting

Reply:
please check